shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -984,6 +992,10 @@ TEST_F(EmbedderTest, VerifyB143464703WithSoftwareBackend) {
   event.width = 1024;
   event.height = 600;
   event.pixel_ratio = 1.0;
+  event.physical_view_inset_top = 0.0;
```

Comment:
It isn't asserted that these values are actually checked in Dart code. I mean, wouldn't this test pass no matter what values you specified here?

Reply:
True, I'll add a better test in this change.

Reply:
You should also add a test with non-zero values.